### DISCUSSION TITLE
Is monai latest version supported in ronin machine?

### DISCUSSION THREAD

**Discussion by Maryam-Mostafa** [Project-MONAI/MONAI#4992] [Q&A] [answered]
is monai version 0.9.1 supported in ronin machine ? when I install the new version of monai like this:
`!pip install -qU "monai[ignite, nibabel, torchvision, tqdm]==0.9.1"`
I also tried this :
`pip install monai==0.9.1`

 but still giving me the old version 0.8.1

**Comment by wyli** [comment]
maybe there are multiple versions installed at different system locations, you can check the file location with 
```
python -c "import monai; print(monai.__file__)"
```

**Comment by wyli** [comment, marked as answer]
perhaps this is the same issue as https://github.com/Project-MONAI/MONAI/discussions/5014, please upgrade python to at least 3.7.

**Comment by Maryam-Mostafa** [comment]
Thanks Wyli 🙏🏻🌸
